### PR TITLE
Add missing licesnse to the Go Driver

### PR DESCRIPTION
**Pull request by prestonvasquez**
NA

## Summary

Run `make add-license`

## Background & Motivation

Two files were missing the Go Driver license, this PR adds those licenses.

### CODE REVIEW

**Review by qingyang-hu — APPROVED** [review]
👍 LGTM!

**Review by benjirewis — CHANGES_REQUESTED** [review]
Our `sa-lint` check [should](https://github.com/mongodb/mongo-go-driver/blob/master/Makefile#L9-L15) be catching the lack of licenses in these files. Any idea why that's not happening? Should we also update the `add-license` target to ignore files that start with `Copied from` (as they already have an external copyright)?

File: mongo/integration/unified/client_encryption_operation_execution.go [hunk 1]
```diff
@@ -1,3 +1,8 @@
+// Copyright (C) MongoDB, Inc. 2022-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
```
Comment (benjirewis) [inline mongo/integration/unified/client_encryption_operation_execution.go:5]:
```suggestion
// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0

```

**Comment by prestonvasquez** [comment]
> Our `sa-lint` check [should](https://github.com/mongodb/mongo-go-driver/blob/master/Makefile#L9-L15) be catching the lack of licenses in these files. Any idea why that's not happening? Should we also update the `add-license` target to ignore files that start with `Copied from` (as they already have an external copyright)?

@benjirewis I've put in a ticket to investigate this, GODRIVER-2675 .

It would probably be better to have a running list of files that we ignore adding a license to, rather than searching for phrases in code.

**Review by benjirewis — APPROVED** [review]
Thanks for filing that. Good point; we may eventually have externally copyrighted files that don't start with `Copied from`. LGTM.

**Pull request merged**